adapters/http-base/src/test/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapterTest.java

```diff
@@ -359,6 +370,158 @@ public void testUploadEventWaitsForAcceptedOutcome() {
                 any());
     }
 
+    /**
+     * Verifies that the adapter waits for an event with an associated command being settled and accepted
+     * by a downstream peer before responding with a 202 status to the device.
+     */
+    @Test
+    public void testUploadEventWithCommandWaitsForAcceptedOutcome() {
```

Comment:
I am not sure if I understand the purpose of this test case compared to the test right before it. The CommandContext does not seem to make any sense in this context anyway, does it? If the device were to respond to a command, then it would not send an event but a command response, wouldn't it?

Reply:
I removed this test and created new ones that should hopefully clearer test the command flow in the HTTP adapter